mlflow/metrics/genai/__init__.py

```diff
@@ -0,0 +1,7 @@
+from mlflow.metrics.genai.metric_definitions import (
+    correctness,
+)
+
+__all__ = [
+    "correctness",
+]
```

Comment:
Is there a reason we need to expose this in mlflow.metrics.genai and mlflow.metrics module as well? Can we remove this?